lib/processors/themeBuilderWorker.js

```diff
@@ -0,0 +1,291 @@
+import workerpool from "workerpool";
+import themeBuilder from "./themeBuilder.js";
+import {createResource} from "@ui5/fs/resourceFactory";
+import {Buffer} from "node:buffer";
+
+/**
+ * Task to build library themes.
+ *
+ * @private
+ * @function default
+ * @static
+ *
+ * @param {object} parameters Parameters
+ * @param {MessagePort} parameters.fsInterfacePort
+ * @param {object[]} parameters.themeResources Input array of Uint8Array transferable objects
+ * 	that are the less sources to build upon. By nature those are @ui5/fs/Resource.
+ * @param {object} parameters.options Less compiler options
+ * @returns {Promise<object[]>} Resulting array of Uint8Array transferable objects
+ */
+export default async function execThemeBuild({
+	fsInterfacePort,
+	themeResources = [],
+	options = {}
+}) {
+	const fsThemeResources = uint8ToResources(themeResources);
+	const fsReader = new FsWorkerThreadInterface(fsInterfacePort);
+
+	const result = await themeBuilder({
+		resources: fsThemeResources,
+		fs: fsReader,
+		options
+	});
+
+	return resourcesToUint8(result);
+}
+
+/**
+ * Casts @ui5/fs/Resource-s into an Uint8Array transferable object
+ *
+ * @param {@ui5/fs/Resource[]} resourceCollection
+ * @returns {Promise<object[]>}
+ */
+export async function resourcesToUint8(resourceCollection) {
```

Comment:
Since we use plain buffers now instead of Uint8Arrays, maybe we should rename these functions?

I'm thinking they might even become standard functions on the Resource class if we need this more often. The alternative is of course to change the themeBuilder implementation to not use Resources but plain objects.

```suggestion
export async function serializeResources(resourceCollection) {
```